web-client/src/app/state/session.service.ts

```diff
@@ -33,6 +35,7 @@ export class SessionService {
     private sessionQuery: SessionQuery,
     private enclaveService: EnclaveService,
     private messagingService: MessagingService,
+    private sessionXrplService: SessionXrplService
```

Comment:
There is a comma missing after `SessionXrplService` 
Hence, the [web-client / test (push)](https://github.com/ntls-io/nautilus-wallet/actions/runs/3445663688/jobs/5749638797) is failing.